### PR TITLE
feat(atenlib): ops 4/n

### PR DESCRIPTION
**Pull request by justinchuby**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #256
* #255
* #252

Some more math and matrix ops

### CODE REVIEW

File: onnxscript/function_libs/torch_aten/ops/core.py [hunk 1]
```diff
@@ -670,16 +670,10 @@ def aten_cdist(
...
 
 
-def aten_ceil(self: TensorType) -> TensorType:
+def aten_ceil(self):
     # ceil(Tensor self) -> Tensor
 
-    raise NotImplementedError()
-
-
-def aten_celu(self: TensorType, alpha: float = 1.0) -> TensorType:
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/core.py:679]: Moved to nn.

File: onnxscript/function_libs/torch_aten/ops/core.py [hunk 2]
```diff
@@ -785,14 +779,6 @@ def aten_clamp_min_tensor(self, min_):
     return op.Max(self, min_)
 
 
-def aten_clip(
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/core.py:788]: Alias of clamp

**Comment by justinchuby** [comment]
@fatcat-z ready for review

**Review by fatcat-z — APPROVED** [review]
LGTM.

**Pull request merged**